File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    const str = s.toLowerCase().split(' ');
    let answer = [];
    
    for (let s of str) {
        const arr = s.split('');
        let word = '';
        for (let i = 0; i < arr.length; i++) {
            if (i === 0) {
                word += arr[i].toUpperCase();
            } else {
                word += arr[i];
            }
        }
        answer.push(word);
    }
    return answer.join(' ');
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    words = s.split(' ')
    answer = []
    
    for word in words:
        tmp = ''
        
        if word:
            tmp = word[0].upper() + word[1:].lower() 
        
        answer.append(tmp)
        
    return ' '.join(answer)